### ISSUE TITLE
"EMFILE: too many open files" while running serverless deploy on Windows

### ISSUE THREAD

**Issue by jerryjeremiah** [serverless/serverless#12735]
### Issue description

When I try to do a serverless deploy on Windows I get "EMFILE: too many open files"

```
C:\ADO\projects\skyfii>npx serverless deploy

Deploying coco-skyfii to stage dev (ap-southeast-2)
Bundling with Webpack...
Package lock found - Using locked versions
Packing external modules: @aws-sdk/client-dynamodb@^3.556.0, @aws-sdk/client-secrets-manager@^3.556.0, @aws-sdk/lib-dynamodb@^3.556.0, source-map-support@^0.5.21

× Stack coco-skyfii-dev failed to deploy (1092s)
Environment: win32, node 20.12.1, framework 3.39.0 (local), plugin 7.2.3, SDK 4.5.1
Credentials: Local, "spark-digitalchannelces-nonprod" profile
Docs:        docs.serverless.com
Support:     forum.serverless.com
Bugs:        github.com/serverless/serverless/issues

Error:
Cannot read file node_modules\@smithy\middleware-retry\dist-types\index.d.ts due to: EMFILE: too many open files, open 'C:\ADO\projects\skyfii\.webpack\updateSecretTokenGCSkyfii\node_modules\@smithy\middleware-retry\dist-types\index.d.ts'
```

I am running node/20.12.1/x64 on Windows 10
I know this wouldn't be a problem on Linux
Do I need to install Linux to deploy this package or is there a way to make it work on Windows?

### Context

[serverless.zip](https://github.com/user-attachments/files/16535796/serverless.zip)

**Comment by Rumirad64** [comment]
a simple restart will fix this many times

**Comment by rahulgandhi2411** [comment]
same issue after restart even i tried to reinstall all packages but same issue, tried increasing page file size but not working

**Comment by skierkowski** [comment]
@jerryjeremiah @rahulgandhi2411 Can you try this on V4 of Serverless Framework? This issue was fixed in a recent release (4.1.12).

**Comment by skierkowski** [comment]
Closing as this issue has been resolved.
